acceptance/bin/diff.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+"""This script implements "diff -r -U2 dir1 dir2" but applies replacements first"""
```

Comment:
Did you have a chance to measure performance difference between running this test with `diff` and with repls.json writing + diff.py?

Reply:
I don't understand -- `diff` is doing the wrong thing there, e.g. it will look at actual username rather than [USERNAME] and detect that as a difference. So performance is not important because they are not equivalent, functionality-wise.

We can bench diff.py against diff but even if it's 2x slower there is no action to take, because we use diff.py for handling replacements.

That's said I don't think diffing will be a bottleneck in your tests, so it makes sense to prefer it because it has less cross-platform quirks than diff.